generator/main.go

```diff
@@ -273,15 +337,182 @@ func generateDoc(destination string) (interface{}, error) {
 	}
 
 	if destination == "Rockset" {
-		doc["_id"] = guuid.New().String()
+		// Monotonic ids
+		doc["_id"] = formatDocId(max_doc_id)
+		max_doc_id = max_doc_id + 1
 	}
 
-	doc["_event_time"] = getCurrentTimeMicros()
+	// Use _ts instead of _event_time as _ts is mutable
+	doc["_ts"] = getCurrentTimeMicros()
 	doc["generator_identifier"] = generatorIdentifier
 
 	return doc, nil
 }
 
+func generatePatches(num_patch int, destination string, c chan map[string]interface{}) ([]interface{}, error) {
+	if destination != "Rockset" {
+		panic("Patches can only be generated for Rockset at this time")
```

Comment:
I asked about using panic in #go channel today - @pmenglund suggested returning error and only using `main()` should use panic